### DISCUSSION TITLE
How to use tailwind inside an iFrame?

### DISCUSSION THREAD

**Discussion by gregorvoinov** [tailwindlabs/tailwindcss#5288] [Help]
How can I use tailwindclasses within an iFrame during development?

Currently, when I add new styles, I'm creating a new css file with
`npx tailwindcss build src/assets/css/tailwind.css -o dist/assets/css/index.css`
and link this file in my iFrame, but this is a bit annoying.... is there a better way to do this during development?

**Comment by Shanky-21** [comment]
when i'm doing `bg-white` in className. its not reflecting in iframe. at the same time its reflecting in elements above iframe. not sure how to fix it. I tried passing `https://cdn.tailwindcss.com"` as a script tag. but it didn't work then also not sure how to proceed.
